src/schnetpack/data/datamodule.py

```diff
@@ -292,6 +331,15 @@ def _load_partitions(self):
                         + " validation partitions need to be set!"
                     )
 
+                # Convert relative sizes to absolute values
```

Comment:
I suggest to move this to the top of the function and replace the `expected_xxx_size` with `self.num_xxx`